src/reducer.js

```diff
@@ -12,8 +12,11 @@ import { QUEUE_ACTION, ONLINE, OFFLINE, RESET_QUEUE } from './actions'
 export default function reducer(state = INITIAL_STATE, action = {}) {
   switch (action.type) {
     case REHYDRATE: { // Handle rehydrating with custom shallow merge.
-      const incoming = action.payload.offline
-      if (incoming) return { ...state, ...incoming }
+
+      if (action.payload && action.payload.offline) {
+        const incoming = action.payload.offline;
+        if (incoming) return { ...state, ...incoming };
```

Comment:
You can remove this if statement since we are certain `action.payload.offline` is present. You can just `return { ...state, ...incoming }` here.

Reply:
Got it!

Reply:
https://github.com/InspireNL/redux-offline-queue/pull/20/commits/8fc2dd7de1dfe44865d23edaff0c99fe79d3c6c7

Done :heavy_check_mark: